### PR TITLE
[miio] adding new devices

### PR DESCRIPTION
**Pull request by marcelrv**
Adding several new devices. including:
yeelink.light.color4 #7016
Mi Home PTX Wall Switches (090615.switch.xswitch03) #7000
Xiaomi Ceiling Lamp (yeelink.light.ceiling11) #6787
Additional sensors and information in zhimi.airpurifier.v7 #5760

Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/database/yeelink.light.color1.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
 			"yeelink.light.color1",
 			"yeelink.light.color2",
 			"yeelink.light.color3",
+            "yeelink.light.color4",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/yeelink.light.color1.json:7]: use tabs to make this consistent with everything else.

File: bundles/org.openhab.binding.miio/README.md [hunk 2]
```diff
@@ -307,7 +315,7 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | Channel          | Type    | Description                         |
 |------------------|---------|-------------------------------------|
 | power            | Switch  | Power                               |
-| mode             | String  | Mode                                |
+| humidifierMode   | String  | Mode                                |
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:318]: Maybe update the description too? The channel name has more information than the description does...

**Comment by marcelrv** [comment]
note, I moved the 2 typo's correction in the readme to the cloud PR o avoid merge conflict.

File: bundles/org.openhab.binding.miio/README.md [hunk 3]
```diff
@@ -2772,12 +2972,28 @@ Color rgbColor "RGB Color" (G_light) {channel="miio:basic:light:rgbColor"}
...
+### Yeelight Color Bulb YLDP02YL 9W (yeelink.light.color3) item file lines
 
 note: Autogenerated example. Replace the id (light) in the channel with your own. Replace `basic` with `generic` in the thing UID depending on how your thing was discovered.
 
 ```java
-Group G_light "Yeelight Color Bulb" <status>
+Group G_light "Yeelight Color Bulb YLDP02YL 9W" <status>
+Switch power "Power" (G_light) {channel="miio:basic:light:power"}
+Number brightness "Brightness" (G_light) {channel="miio:basic:light:brightness"}
+String delayoff "Shutdowm Timer" (G_light) {channel="miio:basic:light:delayoff"}
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/README.md:2983]:
```suggestion
String delayoff "Shutdown Timer" (G_light) {channel="miio:basic:light:delayoff"}
```

File: bundles/org.openhab.binding.miio/src/main/resources/database/yeelink.light.ceiling.json [hunk 4]
```diff
@@ -1,119 +1,123 @@
...
+				"actions": [
+					{
+						"command": "set_bright",
+						"parameterType": "NUMBER"
+					}
+				]
+			},
+			{
+				"property": "delayoff",
+				"friendlyName": "Shutdowm Timer",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/database/yeelink.light.ceiling.json:46]:
```suggestion
				"friendlyName": "Shutdown Timer",
```

**Pull request merged**